pkg/flink/internal/store/store.go

```diff
@@ -168,6 +169,23 @@ func (s *Store) FetchStatementResults(statement types.ProcessedStatement) (*type
 	return &statement, nil
 }
 
+// TODO: remove this when backend fixes latency problem/the RUNNING state is set with the ResultsSchema
```

Comment:
Maybe add the related jira ticket we have?

```suggestion
// TODO: remove this when backend fixes latency problem/the RUNNING state is set with the ResultsSchema
// Related ticket: https://confluentinc.atlassian.net/browse/KFS-1211
```

Reply:
Done!